samples/Mvc.LocalizationSample.Web/CustomValidationAttributeAdapterProvider.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc.DataAnnotations;
+using Microsoft.AspNetCore.Mvc.DataAnnotations.Internal;
```

Comment:
Please open an issue about moving `ValidationAttributeAdapterProvider` and `AttributeAdapterBase<TAttribute>` out of `.Internal`